store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -1133,6 +1133,14 @@ Account getAccountByName(String emailAddress, boolean loadFromMaster, boolean ch
             }
         }
 
+        if (LC.zimbra_additional_logging.booleanValue() && account == null) {
+            String contextString = ZimbraLog.getContextString();
+            // using this clause because other clients like imap, pop get logged though nginx lookup
+            if (contextString != null && contextString.contains("ip")) {
+                ZimbraLog.security.info(String.format("Error occurred during authentication:" +
```

Comment:
we are assuming that this will be auth call but function name is getAccountByName, so internally if we are calling this method then it could give wrong information in logs

Reply:
Thank you for the feedback. I've updated Auth.java to address the logging issue with 'getAccountByName'. Please review the changes and let me know if there are any more concerns